### ISSUE TITLE
Unify options in Change Glyph and Change Weight

### ISSUE THREAD

**Issue by n8willis** [fontforge/fontforge#395]
The Element -> Style -> Change Glyph tool has three tabs, one of which ("Stems") duplicates some, but not all, of the functionality also found in the Element -> Style -> Change Weight tool.  The documentation describes them as offering the same options conceptually, with the bonus that Change Glyphs allows you to simultaneously do Extend/Condense and x-height changes.  Nevertheless, there are specific differences in the options and the results they produce.

Specifically,
(1) Change Weight only permits changing stems by em-units, not by percentages.
(2) Change Weight does not allow changing horizontal and vertical stems by different amounts.
(3) Change Weight includes an option to retain counters or "squish" them, which Change Glyph does not.
(4) Change Weight also offers several more complex modes, like separate LCG and CJK options, plus top/bottom hint "zones", serif detection, and self-intersect detection & cleanup.
(5) Change Weight changes horizontal bearings when transforming a glyph; Change Glyphs doesn't.

In short, there's a LOT of weirdness in these two tools.  The two split up a pretty nice set of options into two places, but in a way that you can't take advantage of all of them -- using either one makes destructive changes to the glyph.  At the very least, (1) and (2) need addressing; whether it makes more sense to duplicate the rest of the options between the two tools or split them so that they don't overlap is a harder question.

There are probably similar weirdnesses in the Horizontal and Vertical tabs of Change Glyphs, since they also correspond to other stand-alone tools in the Style menu, but I haven't messed around with them yet.
